data-prepper-core/src/main/java/org/opensearch/dataprepper/core/logging/SensitiveArgumentMaskingConverter.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.opensearch.dataprepper.logging;
+package org.opensearch.dataprepper.core.logging;
```

Comment:
This package may make sense to leave where it already is. I don't think Log4J is making use of our DI for package scanning.

Reply:
Moved this class back to its original package